src/core/include/iDynTree/Core/Transform.h

```diff
@@ -102,11 +102,25 @@ namespace iDynTree
         Transform(const Rotation & _rot, const Position & origin);
 
         /**
+        * Constructor from a raw row-major ordered buffer of doubles
+        * @param in_data the raw buffer of 16 doubles
+        * @param in_size size of the buffer (must be 16)
+        */
```

Comment:
I would mention that this is equivalent to the `fromHomogeneousTransform` method.

Reply:
Ok! Since it would be possible, do you see any benefit setting as `static` the `fromHomogeneousTransform()`?

Reply:
Another more bindings-friendly possibility is to have this two methods as methods taking in input an `iDynTree::Matrix4x4` classes. This would enable to have in Matlab:
~~~
mat_iDynTree = iDynTree.Matrix4x4();
mat_iDynTree.fromMatlab(eye(4,4)) 
trans_iDynTree = iDynTree.Transform(mat_iDynTree)
~~~
However, we can also add this later, so you can proceed in the way you prefer.